CONFIG/config.yml

```diff
@@ -0,0 +1,54 @@
+network:
+  subnet:
+    first_sheet:
```

Comment:
What does this signify?

Reply:
for a particular network in which we can have multiple subsets, and each subnet can have 2 or more sheets